src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAO.scala

```diff
@@ -11,6 +11,7 @@ import scala.collection.JavaConversions._
 import scala.collection.JavaConverters._
 
 import org.broadinstitute.dsde.rawls.model.AttributeConversions._
+import CachedTypes._
```

Comment:
what happens if we forget to do this?

- does it compile anyway and fail mysteriously? (bad)
- is it obvious that this is the way to fix it?